feature-libs/qualtrics/components/qualtrics-loader/qualtrics-loader.service.ts

```diff
@@ -69,7 +67,7 @@ export class QualtricsLoaderService implements OnDestroy {
 
   constructor(
     protected winRef: WindowRef,
-    protected rendererFactory: RendererFactory2,
+    protected scriptLoader: ScriptLoader,
```

Comment:
To make the automatic migration to work this have to be added as a last constructor parameter.